### DISCUSSION TITLE
Can i remove next-client-pages-loader

### DISCUSSION THREAD

**Discussion by popuguytheparrot** [vercel/next.js#25741] [Help]
Hello everyone. How necessary are these loaders in client chunks and can they be removed? Why are these chunks added to the export?

![image](https://user-images.githubusercontent.com/19804652/120610586-fff13d00-c45b-11eb-9c08-c1b05e70c060.png)

**Comment by M1r1k** [comment]
I am having the same issue, all of my pages show this single lib instead of separate modules this page contains

**Comment by timneutkens** [comment]
This is a bug in webpack bundle analyzer. Does not affect bundle sizes.

**Reply by M1r1k** [reply]
Thanks @timneutkens!
Do you know if there a github issue or discussion related to this problem? Would be nice to find at least temporary workaround :)
If not I can go and create issue in their repo with reproducable example
